### ISSUE TITLE
Allow to stop reading channels and EPG starting from channel X

### ISSUE THREAD

**Issue by M-Reimer** [FernetMenta/vdr-plugin-vnsiserver#115]
Hello,

this is a feature request that came to my mind because of some "technical limitation" of the VDR itself.

One of my friends is using VDR (exclusively with Kodi) and he uses two satellite dishes to have access to english and german channels. This results in several hundred channels in the VDR channels.conf.

Unfortunately VDR doesn't allow to define "favorite lists". Every automatically found channel is placed in the "live" channel list.

Without allowing VDR to auto-add channels the only way is to do this manually by adding to the channels.conf file, which is not user-friendly at all.

The result of this is, that the sync between Kodi and VDR (retrieving of EPG, channel list, ...) takes quite some time.

To clean up the view in Kodi it would really help if VNSI would stop sending data starting with a defined "end channel number" or something like that. Everything after that doesn't appear in Kodi.

**Comment by FernetMenta** [comment]
Why don't you use channel filters? That limits amount of channels to a reasonable number.

**Comment by M-Reimer** [comment]
But then still everything is synced over to Kodi, right?

The idea is, that I can define some "here the shit starts" border.

VNSI should not longer care at all about everything that follows after this (optional) border.

**Comment by FernetMenta** [comment]
> But then still everything is synced over to Kodi, right?

No, vnsiserver only syncs channels that passes the filter. I am talking about VNSI channel filter, not Kodi's channel manager.

**Comment by FernetMenta** [comment]
vnsi channel filters also sort channels.conf. wanted channels are first, then unwanted ones.
